docs/reference/how-to.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[how-to]]
+= How To
+
```

Comment:
Add an intro like:

    [partintro]
    --
    Elasticsearch ships with defaults which are intended to give a good out of
    the box experience. Full text search, highlighting, aggregations, indexing
    should all just work without the user having to change anything.

    Once you better understand how you want to use Elasticsearch, however,
    there are a number of optimizations you can make to improve performance
    for your use case.

    This section provides guidance about which changes should and shouldn't be
    made.
    --